cmd/kubeadm/app/util/marshal.go

```diff
@@ -97,7 +98,7 @@ func SplitYAMLDocuments(yamlBytes []byte) (kubeadmapi.DocumentMap, error) {
 			return nil, err
 		}
 		if len(gvk.Group) == 0 || len(gvk.Version) == 0 || len(gvk.Kind) == 0 {
-			return nil, errors.New("invalid configuration: kind and apiVersion is mandatory information that needs to be specified")
+			return nil, fmt.Errorf("invalid configuration about GroupVersionKind %q: kind and apiVersion is mandatory information that needs to be specified", gvk)
```

Comment:
minor change request:
```
errors.Errorf("invalid configuration for GroupVersionKind %+v: kind and apiVersion is mandatory information that needs to be specified", gvk)
```

`v+%` will print it as `... GroupVersionKind {Group: ..., Version: ..., Kind...}...`